sass/_mixins.scss

```diff
@@ -0,0 +1,12 @@
+@mixin transform($xVal, $yVal: null) {
+  -webkit-transform: translateX($xVal);
+  -webkit-transform: translateY($yVal);
+  transform: translateX($xVal);
+  transform: translateY($yVal);
```

Comment:
this seems a bit repetitive... does this work? I don't actually know if you can pass null.
```scss
@mixin transform($x: null, $y: null) {
  -webkit-transform: translate($x, $y);
  transform: translate($x, $y);
}
```